src/app/states/sleep.jade

```diff
@@ -4,25 +4,29 @@ block input
   md-divider
   h2 {{layout.selected.name}} Input Block
     md-tooltip(md-direction="top") Please, answer few questions to get recommendations right for you
-  div(layout-gt-sm="row",  layout-xs="column", layout-align='space-between center', flex-gt-md="50")
-    md-input-container.md-block.md-accent(flex-gt-sm='60', flex-sm='100')
-      label name
+  div(layout)
+    md-input-container.md-block.md-accent(flex, flex-gt-sm='45')
+      label Name
       input(ng-model='layout.user.name')
 
-  for parameter in ['asleep', 'wake']
-    div(layout='row', layout-align='space-between center', flex-gt-md="30")
-      md-input-container.md-block.md-accent(flex= '20', flex-gt-md='70')
-        label= parameter
-        input(type='time', ng-model='layout.user.'+ parameter, aria-label= parameter)
+  div(layout='column', layout-gt-sm='row', layout-align='space-between', flex-gt-sm='45')
+    md-input-container.md-block.md-accent(flex= '20', flex-gt-sm='45')
+      label Go to sleep at
+      input(type='time', ng-model='layout.user.sleep.'+ parameter, aria-label= parameter)
```

Comment:
what parameter?

hint: try not to mix all the jade features with angular, it makes a mess (you do not always get what is jade variable and what is angular scope data), jade is awesome for extends, includes and syntax